helm/aws-load-balancer-controller/README.md

```diff
@@ -108,10 +108,11 @@ The old controller must be uninstalled completely before installing the new vers
 ### Kubectl installation
 If you had installed the previous version via kubectl, uninstall as follows
 ```shell script
-$ kubectl delete deployment -n kube-system alb-ingress-controller
 # Find the version of the current controller
```

Comment:
We could modify this comment to say 

\# Alternatively you can find the version of the controller and delete as follows
...

Reply:
ok I do it